virtual_rainforest/models/soil/module_schema.json

```diff
@@ -1,28 +1,48 @@
-{
-   "type": "object",
-   "properties": {
-      "soil": {
-         "description": "Configuration settings for the soil module",
-         "type": "object",
-         "properties": {
-            "constants": {
-               "description": "Constants for the soil module",
-               "type": "object",
-               "properties": {
-                  "SoilConsts": {
-                     "type": "object"
-                  }
-               },
-               "required": [
-                  "SoilConsts"
-               ]
-            }
-         },
-         "default": {},
-         "required": []
-      }
-   },
-   "required": [
-      "soil"
-   ]
+{
+   "type": "object",
+   "properties": {
+      "soil": {
+         "description": "Configuration settings for the soil module",
+         "type": "object",
+         "properties": {
+            "constants": {
+               "description": "Constants for the soil module",
+               "type": "object",
+               "properties": {
+                  "SoilConsts": {
+                     "type": "object"
+                  }
+               },
+               "required": [
+                  "SoilConsts"
+               ]
+            },
+            "priority": {
+               "type": "object",
+               "default": {},
+               "properties": {
+                  "init": {
+                     "type": "array",
+                     "default": [],
```

Comment:
Was going to suggest adding default priorities here, but maybe this isn't the best place for them